Frontend/library/src/DataChannel/DataChannelLatencyTestResults.ts

```diff
@@ -19,18 +19,14 @@ export type DataChannelLatencyTestSeq = number;
  * Request sent to Streamer
  */
 export type DataChannelLatencyTestRequest = {
-    Seq: DataChannelLatencyTestSeq;
-    FillResponseSize: number;
-    Filler: string;
+    Seq: DataChannelLatencyTestSeq; FillResponseSize: number; Filler: string;
 }
 
 /**
  * Response from the Streamer
  */
 export type DataChannelLatencyTestResponse = {
-    Seq: DataChannelLatencyTestSeq;
-    Filler: string;
-    ReceivedTimestamp: DataChannelLatencyTestTimestamp;
+    Seq: DataChannelLatencyTestSeq; Filler: string; ReceivedTimestamp: DataChannelLatencyTestTimestamp;
```

Comment:
Odd... they should all get there own line.

Reply:
I cant for the life of me find the setting thats formatting this like this. It seems wild. I assume its because its a type definition being exported, maybe it's tripping up the formatter.